README.md

```diff
@@ -145,6 +145,30 @@ module "egress_space" {
 }
 ```
 
+### egress_proxy
+
+Creates and configures an instance of cg-egress-proxy to proxy traffic from your apps.
+
+Prerequities:
+
+* existing client_space with already deployed apps
+* existing public-egress space to deploy the proxy into
+
+```
+module "egress_proxy" {
+  source = "github.com/GSA-TTS/terraform-cloudgov//egress_proxy?ref=v1.1.0"
```

Comment:
Is the double `/` intentional?

```suggestion
  source = "github.com/GSA-TTS/terraform-cloudgov/egress_proxy?ref=v1.1.0"
```

Reply:
Yes, the double `//` is intentional. [It is the mechanism for packaging multiple modules into a single repo](https://developer.hashicorp.com/terraform/language/modules/sources#modules-in-package-sub-directories)